.github/workflows/update-helm-major-versions.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
 
       - name: Get current date
         id: date
-        run: echo "date=$(date +'%Y-%m-%d')" >> $GITHUB_OUTPUT
+        run: echo "date=$(date +'%Y-%-m-%-d')" >> $GITHUB_OUTPUT
```

Comment:
the date variables here (in both workflows you edited) feed into our dependency update jobs, which are separate from the date value used for the `prefect-helm` chart version

you might want to look over at our `prefect` core repo, which triggers a new helm chart version releases

https://github.com/PrefectHQ/prefect/blob/main/.github/workflows/helm-chart-release.yaml#L15

Reply:
🤦 Thanks for pointing that out, @parkedwards!

I have opened https://github.com/PrefectHQ/prefect/pull/10759